### DISCUSSION TITLE
"TypeError: '>' not supported between instances of 'NoneType' and 'int"

### DISCUSSION THREAD

**Discussion by luisgs** [Flexget/Flexget#3475] [Q&A]
Hi there
After installation and first execution on a new machine, i received this error below. Configuration works in differnt machine

"" flexget check:
  File "/home/luis/.local/lib/python3.10/site-packages/flexget/event.py", line 20, in __call__
    return self.func(*args, **kwargs)
  File "/home/luis/.local/lib/python3.10/site-packages/flexget/db_schema.py", line 170, in upgrade_wrapper
    if new_ver > current_ver:
TypeError: '>' not supported between instances of 'NoneType' and 'int'
""

How to solve it? 
Python 3.10
Flexget 3.3.15

Similar problem is here: https://github.com/Flexget/Flexget/discussions/2887

**Comment by aidan-** [comment]
Can you please add some more information @luisgs ?

- What OS/system are you running this on
- How did you install Flexget
- How are you running Flexget
- Your configuration file

**Comment by paranoidi** [comment]
Did you shutdown or abort initial database creation? Can you try renaming it into something and re-running flexget?

**Comment by ProteinPig** [comment]
You guys can reproduce it as below config

```
    trakt_lookup: yes
    if:
      - trakt_rating > 7.5: accept
      - movie_year < 1990: reject
      - trakt_votes < 50: reject
```

**Reply by gazpachoking** [reply]
This might cause a similar error, but a totally different cause. The OP issues is happening when upgrading the database (and I have no idea how that might happen.) Your issue can be solved by including a default if the lookup fails. i.e.

```yaml
    trakt_lookup: yes
    if:
      - trakt_rating|d(0) > 7.5: accept
      - movie_year|d(0) < 1990: reject
      - trakt_votes|d(0) < 50: reject
```

**Comment by acrane1** [comment]
I am getting the same error. I haven't made any changes to my task in months.
Version info: Flexget 3.5.5 API 1.8.0 Web ui: 2.0.28
All other task are working, this is the only one that stopped working.

    nextclassseason:
      rss:
        url: '******************'  
      all_series:
        season_packs: yes
      content_size:
        max: 27000
      template:
        - qbittorrentnextclassseason
 
The fix for me was just changing the task name, to something new. Re-ran and it's working agian.
[flexgetcrashlog.txt](https://github.com/Flexget/Flexget/files/10153557/flexgetcrashlog.txt)
